frontend/userForm.tsx

```diff
@@ -126,11 +126,13 @@ export function UserForm({id}: {id: number | 'new' | undefined}) {
             break; // updated user
           case "success":
           case "created":
-            history.push(`/manage/user/${data?.data}`)
             unstable_batchedUpdates(() => {
               setNew(false);
+              users.push({email: user.email, id: data.data, name: user.email})
+              setUsers(Object.assign([], users))
               userMessage('User saved', 'success')
             })
+            history.push(`/manage/user/${data?.data}`)
```

Comment:
I would just introduce `const id = data?.data` after `case "created"` and just use this at both locations. Makes it a bit clearer.

Reply:
good point! definitely increases readability!